_data/simple-icons.json

```
@@ -1545,6 +1545,11 @@
             "hex": "4285F4",
             "source": "https://flood.io/"
         },
+        {
+            "title": "Fluentd",
+            "hex": "0E83C8",
+            "source": "https://www.treasuredata.com/opensource/"
```

```suggestion
```

---

Why remove it's will cause schema problem

---

Strange... 
I meant that to suggest replacing the source with https://docs.fluentd.org/quickstart/logo since it is more official (as far as I can see). 

```suggestion
            "source": "https://docs.fluentd.org/quickstart/logo"
```